eclipse-collections-testutils/pom.xml

```diff
@@ -49,17 +49,17 @@
         </dependency>
 
         <!-- External dependencies -->
-
         <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
             <scope>compile</scope>
         </dependency>
 
         <dependency>
-            <groupId>org.junit.jupiter</groupId>
-            <artifactId>junit-jupiter-api</artifactId>
-            <scope>test</scope>
+            <groupId>org.opentest4j</groupId>
+            <artifactId>opentest4j</artifactId>
+            <version>1.3.0</version>
+            <scope>compile</scope>
```

Comment:
VerifyTest class has this - that's what Junit5 throws now.

Reply:
Not sure I understand. You're saying this is an explicit dependency of JUnit5?

Reply:
@Desislav-Petrov, if this is a transitive dependency of JUnit5, but we don't reference any classes in any of the code, then why do we need to include it as an explicit dependency?

Reply:
Ok, I expanded VerifyTest... this is the import:

`import org.opentest4j.AssertionFailedError;`

This is in a test only, not included in the test-utils jar, so why is it not a dependency in the unit tests pom instead?

Reply:
So opentest4j comes transitively from junit5 and the enforcer plugin complains we're using a dependency that's not been explicitly declared:
<img width="892" alt="image" src="https://github.com/user-attachments/assets/7fdadf9a-34cc-40ea-835a-5891e3b56c46" />

Given the build does pass, it might be more sensible to suppress the plugin for this specific library in that module?